newsroom/push/tasks.py

```diff
@@ -13,33 +12,34 @@
 notifier = NotificationManager()
 
 
-@contextmanager
-def locked(_id: str, service: str):
+def get_lock(_id: str, service: str) -> str | None:
     lock_name = f"notify-{service}-{_id}"
     if not lock(lock_name, expire=300):
         logger.debug(f"Lock conflict on {lock_name}")
-        return
-
-    logger.debug("Starting task %s", lock_name)
-    try:
-        yield lock_name
-    finally:
-        unlock(lock_name)
-        logger.debug("Done with %s", lock_name)
+        return None
+    return lock_name
 
 
 @celery.task
 async def notify_new_wire_item(_id, check_topics=True):
-    with locked(_id, "wire"):
+    lock = get_lock(_id, "wire")
```

Comment:
nit: I would suggest using a different variable name as we also import `lock` from superdesk
```suggestion
    wire_lock = get_lock(_id, "wire")
```